Add `Resurrection Cup  2026: Registrations Now Open!` news post (#14716)
ppy/osu-wiki: news/2026/2026-05-11-resurrection-cup-2026-registrations-now-open.md

| Screening phase | 2 June – 8 June |
| Qualifier stage | 12 June – 14 June |
| Round of 32 | 19 June – 21 June |
| Round of 16 | 26 June – 28 June |
| Quarterfinals | 3 July – 5 July |
| Semifinals | 10 July – 12 July |
| Finals | 17 July – 19 July |
| COE break | 24 July - 2 August |
| Grand Finals | 7 August – 9 August |

## How do I register?

To register, grab at least 3 friends and fill out the [registration form](https://docs.google.com/forms/d/e/1FAIpQLSdxGhsO75C87lybcdrjtYsYChhmxRmPRxtaWCe321n1eFmBLw/viewform) to sign up as a team! Before registering, please make sure that:

- You have not have violated the [osu! community rules](/wiki/Rules) within the past 12 months.
- You have joined the [ResCup26 Discord server](https://discord.com/invite/UNzyfgGfeu).

We also encourage spectators to join the [Discord server](https://discord.com/invite/UNzyfgGfeu) in order to keep up with important announcements and livestream notifications. For more information, check out the [forum post](https://osu.ppy.sh/community/forums/topics/2205168) and [main sheet](https://docs.google.com/spreadsheets/d/1Md62htYhsmBaVn8eM29XJBgaFCbmvcLJyqdcu5k9Gwo).

> 

Reviewer: ```suggestion

---

```

---
layout: post
title: "Resurrection Cup 2026: Registrations Now Open!"
date: 2026-05-11 12:00:00 +0000
series: community_tournaments
---

> We’re back!

Reviewer: `’` -> `'` everywhere ty

Author: Right, overlooked those, thanks!

![](/wiki/shared/news/2026-05-11-resurrection-cup-2026-registrations-now-open/banner.jpg)

Resurrection Cup, or ResCup for short, is an annual open-rank 3v3, team size 6 tournament that seeks to highlight the pinnacle of the osu!standard tournament scene.

## What’s new?

In line with this year’s theme, "Codename: Genesis", the ResCup team has reoriented its philosophy towards the "genesis" of tournament success – the players. Hence, the 5th iteration of ResCup brings drastic changes to its internal management.

> Not to worry, ResCup will still deliver on its yearly promise to provide high-level production for its players, but the way in which the means reach the ends has been heavily reworked. These include tighter controls on quality across all fronts — from originals to mapping to design to broadcast — a more streamlined development and quality assurance cycle, and a player-centric pooling philosophy.

Reviewer: ```suggestion
Not to worry, ResCup will still deliver on its yearly promise to provide high-level production for its players, but the way in which the means reach the ends has been heavily reworked. These include tighter controls on quality across all fronts — originals, mapping, design and broadcast — a more streamlined development and quality assurance cycle, and a player-centric pooling philosophy.
```

Author: Applied in 5c7afa941b937731a3a5da6ec9703f2ce20471d9 Alongside all the other suggestions below.

> As such, this iteration, we have heavily cut back our originals count this year to 18, essentially halving the scale at which we operated at from last year. This change has allowed us to more efficiently allocate resources, alleviating the burden on poolers and designers, and allowing each original to shine just a little bit brighter.

Reviewer: this iteration and this year

```suggestion
As such, we have heavily cut back our originals count this year to 18, essentially halving the scale at which we operated at from last year. This change has allowed us to more efficiently allocate resources, alleviating the burden on poolers and designers and allowing each original to shine just a little bit brighter.
```

In short, as this year’s ResCup takes a step back from previous iterations in terms of scale, it takes two steps forward in terms of efficiency and quality while keeping what truly matters at heart. Take this news post as our hope that this ResCup will top its predecessors in terms of quality over quantity.

## Tournament schedule

| Event | Dates |
| --: | :-- |
| Registration phase | 11 May – 2 June (0 UTC) |
| Screening phase | 2 June – 8 June |
| Qualifier stage | 12 June – 14 June |
| Round of 32 | 19 June – 21 June |
| Round of 16 | 26 June – 28 June |
| Quarterfinals | 3 July – 5 July |
| Semifinals | 10 July – 12 July |
| Finals | 17 July – 19 July |

> | COE Break | 24 July - 2 August |

Reviewer: randomly in title case

```suggestion
| COE break | 24 July - 2 August |
```

| Grand Finals | 7 August – 9 August |

## How do I register?

To register, grab at least 3 friends and fill out the [registration form](https://docs.google.com/forms/d/e/1FAIpQLSdxGhsO75C87lybcdrjtYsYChhmxRmPRxtaWCe321n1eFmBLw/viewform) to sign up as a team! Before registering, please make sure that:

- You have not have violated the [osu! community rules](/wiki/Rules) within the past 12 months.

> - You have joined the [ResCup26 Discord server](https://discord.gg/UNzyfgGfeu).

Reviewer: ```suggestion
- You have joined the [ResCup26 Discord server](https://discord.com/invite/UNzyfgGfeu).
```

If you don't have enough team members, try the `#looking-for-team` channel in the [Discord server](https://discord.com/invite/UNzyfgGfeu) to find free agents, or fill in this [free agent form](???) to sign up as a free agent yourself! Do note that free agent teams will not be created by the tournament administration.

We also encourage spectators to join the [Discord server](https://discord.com/invite/UNzyfgGfeu) in order to keep up with important announcements and livestream notifications. For more information, check out the [forum post](???) and [main sheet](https://docs.google.com/spreadsheets/d/1Md62htYhsmBaVn8eM29XJBgaFCbmvcLJyqdcu5k9Gwo).

> One last thing, all of our originals for [Resurrection Cup 2025](/wiki/Tournaments/RESC/2025) have been added to the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/462)!

Reviewer: ```suggestion
One last thing: all of our originals for [Resurrection Cup 2025](/wiki/Tournaments/RESC/2025) have been added to the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/462)!
```

We hope to see you around!

> – gamicrayola

Reviewer: ```suggestion
—gamicrayola
```
